osu.Game.Rulesets.Osu/UI/OsuResumeOverlay.cs

```diff
@@ -29,16 +29,25 @@ public class OsuResumeOverlay : ResumeOverlay
         [BackgroundDependencyLoader]
         private void load()
         {
-            Add(clickToResumeCursor = new OsuClickToResumeCursor { ResumeRequested = Resume });
+            Add(cursorScaleContainer = new Container
+            {
+                RelativePositionAxes = Axes.Both,
+                Child = clickToResumeCursor = new OsuClickToResumeCursor { ResumeRequested = Resume }
+            });
         }
 
         public override void Show()
         {
             base.Show();
-            clickToResumeCursor.ShowAt(GameplayCursor.ActiveCursor.Position);
+            cursorScaleContainer.MoveTo(GameplayCursor.ActiveCursor.Position);
+            clickToResumeCursor.Appear();
 
             if (localCursorContainer == null)
+            {
                 Add(localCursorContainer = new OsuCursorContainer());
+
+                localCursorContainer.CursorScale.BindValueChanged(scale => cursorScaleContainer.Scale = new Vector2(scale.NewValue), true);
```

Comment:
why is this required? shouldn't the cursor container be handling this itself?

Reply:
okay i just misread because this is really not the right way to use bindables. you can't `BindValueChanged` to a bindable owned by another class. make a local one.

Reply:
I'm not sure what you mean by that. Do you want me to make a local bindable that's bound to the external one and subscribe to that instead?

Reply:
I think local copy bound to external is correct here - as far as I understand subscribing to the external directly is a memory leak once this particular subscriber goes out of scope.